### PR TITLE
Use accept-language.

### PR DESCRIPTION
**Pull request by kaj**
Set accept-language on all backend requests in frontent, and propagate that header to any requests to social in the backend.

This fixes the translations for service links.

### CODE REVIEW

File: kpm/kpm-backend/src/session.ts [hunk 1]
```diff
@@ -65,9 +65,12 @@ export async function sessionApiHandler(
...
     if (user) {
-      let notes = await getSocial<NumNotes>(user, "notifications/count").catch(
-        socialErr
-      );
+      const lang = req.headers["accept-language"];
+      let notes = await getSocial<NumNotes>(
+        user,
+        "notifications/count",
+        lang
+      ).catch(socialErr);
```
Comment (jhsware) [inline kpm/kpm-backend/src/session.ts:73]: Is this endpoint language aware? Or are you passing lang for consistency?

**Review by jhsware — CHANGES_REQUESTED** [review]
Do we need to look at KOPPS-calls too?

File: kpm/kpm-backend/src/session.ts (hunk 1 above)
Comment (kaj) [reply kpm/kpm-backend/src/session.ts:73]: Just for consistency.  Currently, that endpoint only returns an object-wrapped json number.

**Comment by kaj** [comment]
The kopps-dependant things we cache are independent of language (contains both languages), and I think we should keep it that way.  So no, I don't think we should change our kopps calls. But possibly our backend calls (which use those cached values) ?

File: kpm/kpm-backend/src/session.ts (hunk 1 above)
Comment (kaj) [reply kpm/kpm-backend/src/session.ts:73]: (I made the argument required, to avoid missing it anywhere it might be needed).

**Comment by kaj** [comment]
Den kopps-endpoint vi använder i nuläget (`${KOPPS_API}/course/${course_code}/courseroundterms`) ger språkobjekt {sv: ..., en: ...}, så det är såna vi cachar.  Så inga problem där.  Vi skulle ju kunna tolka de objekten i kpm-backend och bara skicka rätt språk till kpm-frontend, men jag tycker nog att det kan vara en separat fråga till någon annan gång ...

**Comment by jhsware** [comment]
LGTM

**Review by jhsware — APPROVED** [review]
LGTM

**Pull request merged**